Database/Corrections/Automatic/sodBaseQuests.lua

```diff
@@ -399,6 +399,17 @@ function SeasonOfDiscovery:LoadBaseQuests()
             [questKeys.requiredRaces] = raceIDs.NONE,
             [questKeys.requiredClasses] = classIDs.NONE,
         },
+        [78270] = {
+            [questKeys.name] = "Goblin Lockpicks",
+            [questKeys.startedBy] = {{210187}},
+            [questKeys.finishedBy] = {{4086}},
+            [questKeys.requiredLevel] = 1,
+            [questKeys.questLevel] = 23,
+            [questKeys.requiredRaces] = raceIDs.NONE,
+            [questKeys.requiredClasses] = classIDs.NONE,
```

Comment:
```suggestion
            [questKeys.requiredClasses] = classIDs.ROGUE,
```

Reply:
Yeah, last night the scraper and WoWHead had barely any information on it, and didn't include that it was a Rogue quest.

Fixed in: https://github.com/Questie/Questie/pull/5339/commits/3197ef226acfa58c0d5e76bd3c75d0cb3cb9fd5f

Added in the `zoneOrSort` in `sodQuestFixes.lua` here: https://github.com/Questie/Questie/pull/5339/commits/2b8bf89b8d9637a47ab351d5eb1809d9f6728214